electrumsv/wallet.py

```diff
@@ -5034,6 +5040,42 @@ async def _consume_tip_filter_matches_async(self, state: ServerConnectionState)
             * If the batch is processed then it is no longer valid.
             * If the batch is processed (task cancelled) then it will be picked up next restart.
         """
+        async def add_missing_transactions_from_db() -> None:
```

Comment:
All the changes to this function or even file should be able to be reverted. See the comment for functions.py

Reply:
I think it's important to have an initial call of `add_missing_transactions_from_db()` on first launch of this async task... 
but the ignoring of subsequent pushdata notifications can indeed be done in `functions.py` as per your preference.

Reply:
See my other comment.